pyproject.toml

```diff
@@ -149,7 +149,7 @@ select = [
   "B", "B904",   # flake8-bugbear
   "I",           # isort
   "ARG",         # flake8-unused-arguments
-  "C4",          # flake8-comprehensions
+  "C40",         # flake8-comprehensions
```

Comment:
`C4` is actually better, you can ignore the warning, it will be removed in the next version.

Reply:
Thanks for letting me know. I will revert this change here and in Particle.